### DISCUSSION TITLE
[Feature Request] Mark PR as merged from Commit Message

### DISCUSSION THREAD

**Discussion by Disservin** [community/community#86524] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

It should be possible to mark Pull Requests as merged from commits made to repository.
Currently one only has access to things like `close`, `closes`, `resolves` but these keywords, mark the PR as closed, not as merged.

Example:

```
This adds a new app feature

merges #2144
```

**Comment by OJFord** [comment]
Duplicate of #6414
